### DISCUSSION TITLE
MySQL에서 위치(Point) 데이터 타입 사용

### DISCUSSION THREAD

**Discussion by scarf005** [wanted-7h/hotplace#46] [Ideas]
- https://sequelize.org/api/v6/class/src/data-types.js~geometry
- https://dev.mysql.com/doc/refman/8.0/en/spatial-convenience-functions.html
- https://stackoverflow.com/questions/12504208/what-mysql-data-type-should-be-used-for-latitude-longitude-with-8-decimal-places
- https://dev.mysql.com/doc/refman/8.0/en/spatial-type-overview.html

**Comment by Yonge2** [comment]
유용할 것 같네요! 변경할 때 참고하겠습니다
